packages/codd/tests/test_codd_cli.py

```
@@ -295,3 +297,277 @@ def test_validate_exit_codes(tmp_path) -> None:
     (tmp_path / "docs/design.md").unlink()
     _write(tmp_path, "docs/lonely.md", _doc("design:lonely", "design"))
     assert cli.cmd_validate(tmp_path, _config()) == 0
+
+
+# ---------------------------------------------------------------------------
+# impact 分析（compute_impact: pure ロジック）
```

Ruff (0.15.18)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)